File: zb1/codingTest/220317/o/(1)OlympicPlayerNumber/OlympicPlayerNumber.py
def solution(arr, num):
    """
    :param arr: int[][]
    :param num: int
    :return: int
    """

    mergedArr = []
    for eachArr in arr:
        mergedArr += eachArr
    mergedArr.sort()
    return mergedArr[num - 1]

File: zb1/codingTest/220317/o/(1)OlympicPlayerNumber/OlympicPlayerNumber.js
/**
 * @param arr {number[][]}
 * @param num {number}
 * @returns {number}
 */
function solution(arr, Num) {
  let list = [];
  for (let i = 0; i < arr.length; i++) {
    for (let j = 0; j < arr[i].length; j++) {
      list.push(arr[i][j]);
    }
  }
  list.sort(function (x, y) {
    return x - y;
  });
  return list[Num - 1];
}

console.log(
  solution(
    [
      [1, 2, 22],
      [3, 4, 23],
      [8, 9, 33],
    ],
    7
  )
);
